src/server/handlers/webhook.js

```diff
@@ -64,6 +64,7 @@ export const notify = (req, res) => {
     // XXX cjwatson 2017-02-16: We could be smarter about this by looking at
     // the content of the push event.
     return getMemcached().del(getSnapNameCacheId(repositoryUrl))
+      .catch(() => undefined)
```

Comment:
As mentioned earlier - instead of silently ignoring error and returning undefined I'd prefer at least logging the error.